LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthResultInfo.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.BrowserSwitchFinalResult
+
+/**
+ * Details of a [LocalPaymentAuthResult.Success]
+ */
+class LocalPaymentAuthResultInfo internal constructor
```

Comment:
I believe detekt will enforce having the open parenthesis on this line. 

You can run the `./ci lint` command locally to run all of the lint checks.